.github/workflows/autofix.yml

```diff
@@ -0,0 +1,32 @@
+name: autofix.ci
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  autofix:
+    name: autofix
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' || !github.event.pull_request.draft }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - uses: oven-sh/setup-bun@v2
+        with:
+          bun-version: '1.1.39'
+      - run: bun install
+      - run: bun run format:fix
+      - run: bun run lint:fix
```

Comment:
Is it a good idea to add `bun run editorconfig-checker -format github-actions` introduced in #4321?

Reply:
From what I can see `editorconfig-checker` only reports issues in PRs, it doesn't appear to have the ability to fix those issues 👀

Reply:
Ah, you are right! So, it's not needed.